### ISSUE TITLE
Finto-liitännäinen Metatietosanastosta

### ISSUE THREAD

**Issue by AnneliO** [KohaSuomi/Koha#373]
### Kehitysehdotukseni

Voisiko tehdä Finto-liitännäisen, joka poimii termejä [Finton Metatietosanastosta](https://finto.fi/mts/fi/)?

Olisiko liitännäistä mahdollista kohdistaa/tarkentaa sillä tavalla, että jossakin kentässä se hakisi valittavat termit vain yhdestä tai muutamasta Metatietosanaston hierarkian alahaarasta esim. [Kohderyhmä](https://finto.fi/mts/fi/page/m176), [Nimeke](https://finto.fi/mts/fi/page/m1803) tai [Manifestaation suhteet manifestaatioon](https://finto.fi/mts/fi/page/m373)?

Esim. kenttään 246i pitäisi saada tarjolle metatietosanastosta [Nimeke-termin](https://finto.fi/mts/fi/page/m1803) alla olevat vaihtoehdot:

aiempi päänimeke
alanimeke
alanimeke esiössä
alanimeke irtopäällyksessä
alanimeke kannessa
alanimeke selässä
alanimeke toimitustiedoissa

Tällä tavalla valikkoon tulisi tulevaisuudessa tarjoille sitten lisääkin termejä, jos niitä lisättäisiin sanastoon.

### Jotain muuta?

Toive tullut Antti Heikkiseltä.

**Comment by lmstrand** [comment]
Ainakin alustavan tutkimuksen mukaan rajapinta tukee mts-hakuja ja noita "alatunnusten" käyttöä. Jos ei suoraan, niin parsimalla sitten perushausta? Ei ehkä minuutin, mutta parhaimmillaan kahden minuutin homma.

**Comment by AnneliO** [comment]
Koha-Suomen asiantuntijaryhmä 5.2.2024: Toteutetaan.

Huomioitava, että nämä liitännäiset eivät välttämättä toimi Bibframessa.

**Comment by lmstrand** [comment]
Mitä muita käyttötapauksia pluginille olisi? Missä kaikissa kentissä käytettäisiin, ja minkä termin alla olevat tiedot/valittavat vaihtoehdot niissä kentissä haettaisiin valittavaksi?
Olisiko tarkoitus siis suorittaa haku termien alla olevista vaihtoehdoista, vaiko listata kaikki tarjolla olevat vaihtoehdot? Miten esim. kaksi metatietosanaston hierarkian alahaaran tulosjoukkoa yhdistettäisiin?

**Comment by AnttHeik** [comment]
Hei, Metatietosanastoon on tulossa tämän vuoden aikana paljon uudistuvaan RDA:han liittyviä muutoksia, jotka saattavat vaikuttaa myös sanaston rakenteeseen/hierarkoihin. Mietimme sen takia, että jos jätetään tämä vielä odottamaan ja palataan syksyllä asiaan, kun tiedetään tarkemmin, mitä kaikkea muuttuu. Sitten on helpompi ohjeistaa pluginin teossakin.

**Comment by AnneliO** [comment]
Joko Metatietosanaston muutokset on selvinnyt?

**Comment by AnttHeik** [comment]
Joo, metatietosanasto on nyt päivitetty, mutta kaikkia muutoksia ei ole vielä virallisesti otettu käyttöön, koska sovellusohjeiden tekeminen on vielä kesken. Tätä voisi alkaa kyllä pohtimaan taas, missä kentissä tätä voisi jo käyttää, kunhan vain ehtisi jossain välissä :)

**Comment by AnneliO** [comment]
Tämä odottaa määrityksiä.
